cmd/integrations.go

```diff
@@ -0,0 +1,136 @@
+package cmd
+
+import (
+	"github.com/urfave/cli"
+
+	"github.com/Scalingo/cli/cmd/autocomplete"
+	"github.com/Scalingo/cli/integrations"
+)
+
+var (
+	IntegrationsListCommand = cli.Command{
+		Name:     "integrations",
+		Category: "Integrations",
+		Usage:    "List your external integrations",
+		Description: `List all the external integrations associated with your account:
+
+	$ scalingo integrations
+
+	# See also commands 'integrations-create', 'integrations-destroy', 'integrations-import-keys'`,
+
+		Action: func(c *cli.Context) {
+			err := integrations.List()
+			if err != nil {
+				errorQuit(err)
+			}
+		},
+		BashComplete: func(c *cli.Context) {
+			_ = autocomplete.CmdFlagsAutoComplete(c, "integrations")
+		},
+	}
+
+	IntegrationsCreateCommand = cli.Command{
+		Name:     "integrations-create",
+		Category: "Integrations",
+		Flags: []cli.Flag{appFlag,
+			cli.StringFlag{Name: "url", Usage: "URL of the integration", Value: "<url>", EnvVar: ""},
+			cli.StringFlag{Name: "token", Usage: "Token of the integration", Value: "<token>", EnvVar: ""},
+		},
+		Usage: "Create a link between external integration and your account",
+		Description: `Create a link between external integration and your account:
+
+	For github.com and gitlab.com:
+	$ scalingo integrations-create github
```

Comment:
Don't you need the `--token` option for this integration type?

Reply:
The CLI displays an URL for github.com and gitlab.com, to make an OAuth dance, this OAuth dance create the SCM integrations automatically. The token is useless for this.